lib/helpers.rb

```diff
@@ -129,11 +129,17 @@ def handle_threads_query(
     sort_key,
     sort_order,
     page,
-    per_page
+    per_page,
+    context=:course
```

Comment:
Nit: rather than track down every call to `handle_threads_query`, is there a way to set a default value for this parameter?